content/docs/user-guide/data-management/remote-storage/azure-blob-storage.md

```diff
@@ -219,3 +219,11 @@ See `dvc remote modify` for more command usage details.
 
 [Azure documentation]:
   https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob#other-client--per-operation-configuration
+
+- `exclude_environment_credential` - bool. If `true`, excludes the environment credential source for Azure Remote.
+
+- `exclude_visual_studio_code_credential` - bool. If `true`, excludes Visual Studio Code credential source for Azure Remote.
```

Comment:
`bool` is not needed (see how we do other items - e.g. we don't specify `int` for the connection_timeout`.